src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/DeferredResolver.scala

```diff
@@ -19,6 +20,7 @@ class DeferredResolver extends sangria.execution.deferred.DeferredResolver[Consi
       case DeferParentFolder(consignmentId) => context.consignmentService.getConsignmentParentFolder(consignmentId)
       case DeferConsignmentSeries(consignmentId) => context.consignmentService.getSeriesOfConsignment(consignmentId)
       case DeferConsignmentBody(consignmentId) => context.consignmentService.getTransferringBodyOfConsignment(consignmentId)
+      case DeferFile(consignmentId) => context.fileMetadataService.getFileMetadata(consignmentId)
```

Comment:
Should this be plural? `DeferFiles`? Otherwise the name makes it sound like you fetch one file at once, but it actually returns a list.

Reply:
Yep, that makes sense. I've changed it.